extensions/adobe/experience/campaign/feedbackevent.schema.json

```diff
@@ -0,0 +1,244 @@
+                },
+                "xdm:type": {
+                  "title": "Type",
+                  "type": "string",
+                  "description": "Whether this message message event describes the BCC, CC or To recipient."
+                }
+              }
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/messageFailure": {
+          "title": "Message Failure",
+          "type": "object",
+          "properties": {
+            "xdm:canonicalReason": {
+              "title": "Canonical Reason",
+              "type": "string",
+              "description": "Canonicalized text of the response returned by the remote server due to a failed delivery attempt."
+            },
+            "xdm:reason": {
+              "title": "Reason",
+              "type": "string",
+              "description": "Unmodified, exact response returned by the remote server due to a failed delivery attempt."
+            },
+            "xdm:errorCode": {
+              "title": "Error Code",
+              "type": "string",
+              "description": "Error code by which the remote server described a failed delivery attempt."
+            },
+            "xdm:bounceType": {
+              "title": "Bounce Type",
+              "type": "string",
+              "description": "Classifies whether it is a soft bounce or hard bounce."
+            },
+            "xdm:bounceClass": {
+              "title": "Bounce Class",
+              "type": "string",
+              "description": "Classification code for a given message."
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/solutionContext": {
+          "title": "Solution Context",
+          "type": "array",
+          "description": "List of solution specific key value pairs.",
+          "items": {
+            "$ref": "https://ns.adobe.com/experience/analytics/keyvalue"
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/channel": {
+          "title": "Delivery channel",
+          "type": "object",
+          "properties": {
+            "xdm:type": {
+              "title": "Channel Type",
+              "type": "string",
+              "description": "Classifies whether the channel is email, push, sms."
+            },
+            "https://ns.adobe.com/experience/campaign/email": {
+              "title": "E-mail Channel",
+              "type": "object",
+              "properties": {
+                "https://ns.adobe.com/experience/campaign/interactions": {
+                  "title": "User Interactions",
+                  "type": "object",
+                  "properties": {
+                    "xdm:fbType": {
+                      "title": "FB Type",
+                      "type": "string",
+                      "description": "Type of spam report entered against this message (see RFC 5965 § 7.3)."
+                    },
+                    "xdm:reportBy": {
```

Comment:
reportedBy

Reply:
This field is specific to the email channel. If someone has reported an email as spam, then this field would capture email address of that person. It is a type of user interaction with the email message. Hence, it is under "https://ns.adobe.com/experience/campaign/email->https://ns.adobe.com/experience/campaign/interactions".

Reply:
Done.